bindings/java/c/host.c

```diff
@@ -36,149 +36,122 @@ static jbyteArray CopyDataToJava(JNIEnv* jenv, const void* ptr, size_t size)
 
 static bool account_exists_fn(struct evmc_host_context* context, const evmc_address* address)
 {
-    bool result = false;
     const char java_method_name[] = "account_exists";
     const char java_method_signature[] = "(I[B)I";
+
     assert(context != NULL);
     JNIEnv* jenv = attach();
 
-    jclass host_class;
-    jmethodID method;
-    jint jcontext_index;
-    jbyteArray jaddress;
-
     // get java class
-    host_class = (*jenv)->FindClass(jenv, "org/ethereum/evmc/Host");
+    jclass host_class = (*jenv)->FindClass(jenv, "org/ethereum/evmc/Host");
     assert(host_class != NULL);
 
     // get java method
-    method = (*jenv)->GetStaticMethodID(jenv, host_class, java_method_name, java_method_signature);
+    jmethodID method =
+        (*jenv)->GetStaticMethodID(jenv, host_class, java_method_name, java_method_signature);
     assert(method != NULL);
 
     // set java method params
-    jcontext_index = context->index;
-    jaddress = CopyDataToJava(jenv, address, sizeof(struct evmc_address));
+    jbyteArray jaddress = CopyDataToJava(jenv, address, sizeof(struct evmc_address));
 
     // call java method
-    jint jresult = (*jenv)->CallStaticIntMethod(jenv, host_class, method, jcontext_index, jaddress);
-    result = !!jresult;
-
-    return result;
+    jint jresult = (*jenv)->CallStaticIntMethod(jenv, host_class, method, context->index, jaddress);
+    return jresult == 1;
```

Comment:
```suggestion
    return jresult != 0;
```
More classic "convert int to bool".

Reply:
Fixed.